packages/generator-langium/templates/web/langium-config.json

```diff
@@ -8,7 +8,7 @@
             "out": "syntaxes/<%= language-id %>.tmLanguage.json"
         },
         "monarch": {
-            "out": "syntaxes/<%= language-id %>.monarch.ts"
+            "out": "src/syntaxes/<%= language-id %>.monarch.ts"
```

Comment:
Interesting that this file moves. I was wondering why it was under /syntaxes and not under /src

Reply:
Me, too when I looked at it again, because it adds non-required complexity. It is ok to have it in the main src directory, I think